### DISCUSSION TITLE
After my graduation at the universiy, what happen to my account?

### DISCUSSION THREAD

**Discussion by Francesco0610** [community/community#77199] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What I should do with my student account now that I'm graduated, I don't want to lose all my projects, is there a way that I can do? Maybe creating a new account and connect them. Is this possible?

**Comment by ghost** [comment]
This is a small solution for it xd.

(Forgive me if I don't explain myself correctly :"v)

you could download all your projects.

and then organize your new github account.

You create again the repositories where you had your projects saved.

and you decompress the downloaded files, use the git tool and add all your files from the previously downloaded and decompressed documents.

obviously you will move the files to each corresponding repository xd

👍

second solution: 

change the owner of the repositories and add your original account. 

It is in the repository settings (Danger Zone).
